gbpservice/tests/contrib/diskimage-create/elements/nfp-reference-configurator/install.d/99-nfp-service

```diff
@@ -1,22 +1,36 @@
 #! /bin/bash
 set -eu
 
-# copy the reference_configurator folder to VM at /home/ubuntu/
-mv /root/reference_configurator /home/ubuntu/
-ls -lR /home/ubuntu/
+# copy the reference_configurator and pecan_api folders to VM at 
+# /usr/local/lib/python2.7/dist-packages/
+
+cd /usr/local/lib/python2.7/dist-packages/
+
+service_path='gbpservice/tests/contrib/nfp_service'
+pecan_path='gbpservice/nfp'
+mkdir -p $service_path
+mkdir -p $pecan_path
+
+find gbpservice/ -type d -exec touch {}/__init__.py \;
```

Comment:
Don't touch this file. Do actual copy. In case something gets added to this file in future which is needed here, it will break.